libs/sdk-ui-kit/styles/scss/Form/_mixins.scss

```diff
@@ -32,8 +32,6 @@
     font-size: 14px;
     font-family: variables.$gd-font-primary;
     font-weight: 200;
-
-    @include mixins.transition(all, 0.25s, ease-in-out);
```

Comment:
This needs to be removed, otherwise custom outline color blinks from original browser color.